Gruntfile.js

```diff
@@ -0,0 +1,35 @@
+module.exports = function(grunt) {
+
+  // Project configuration.
+  grunt.initConfig({
+    pkg: grunt.file.readJSON('package.json'),
+    uncss: {
+      dist: {
+        options: {
+          stylesheets: [
+            'assets/css/styles.css'
+          ]
+        }
+        files: {
```

Comment:
Expected '}' to match '{' from line 7 and instead saw 'files'.<br>Expected '}' to match '{' from line 6 and instead saw ':'.<br>Expected '}' to match '{' from line 4 and instead saw '{'.